### PR TITLE
Security/privacy questionnaire for unsigned bundles

### PR DESCRIPTION
**Pull request by jyasskin**
I haven't double-checked all the answers, and this needs some more links. I'll go over it again tomorrow.

### CODE REVIEW

File: explainers/navigation-to-unsigned-bundles.md [hunk 1]
```diff
@@ -237,6 +230,115 @@ in the bundle, there are several options:
...
+None.
+
+#### 7. Does this specification allow an origin access to sensors on a user’s device
+
+No.
+
+#### 8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+Network requests (e.g. fetch or iframe) from the unsigned bundle could expose IP
+address of the user, in the same way as regular navigation does.  See also this
```
Comment (kinu) [inline explainers/navigation-to-unsigned-bundles.md]: https://github.com/WICG/webpackage/blob/master/explainers/navigation-to-unsigned-bundles.md#network-access

File: explainers/navigation-to-unsigned-bundles.md [hunk 2]
```diff
@@ -237,6 +230,115 @@ in the bundle, there are several options:
...
+section in the explainer about how network requests from untrusted bundles
```
Comment (kinu) [inline explainers/navigation-to-unsigned-bundles.md]:
```suggestion
[section](#network-access) in the explainer about how network requests from untrusted bundles
```

File: explainers/navigation-to-unsigned-bundles.md [hunk 3]
```diff
@@ -237,6 +230,115 @@ in the bundle, there are several options:
...
+
+See the security considerations in the format spec, security and privacy
+considerations in the explainer, and open design questions in the explainer.
+
+#### 16. Does this specification allow downgrading default security characteristics?
+
+Similarly to Service Worker, this specification allows a resource in a “trusted”
+Web Bundle to be served as its claimed URL if the claimed URL matches within the
+scope of “unsigned bundle scope” of the distributor URL, and
+Service-Worker-Allowed response header also allowed the scope to be extended
```
Comment (kinu) [inline explainers/navigation-to-unsigned-bundles.md]:
```suggestion
if `Service-Worker-Allowed` response header is given the default scope can be also extended```

The original text was probably a bit garbled.. (feel free to modify further)

File: explainers/navigation-to-unsigned-bundles.md [hunk 4]
```diff
@@ -237,6 +230,115 @@ in the bundle, there are several options:
...
+
+#### 2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes.
+
+#### 3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+When the content included in the unsigned Web Bundle is loaded as an untrusted
+bundle (e.g: loaded a “untrusted” bundle from a local file), navigation to the
+Bundle will be loaded within its unique origin with package: scheme, which is
```
Comment (horo-t) [inline explainers/navigation-to-unsigned-bundles.md]:
```suggestion
Bundle will be loaded within its unique origin with [package: scheme](#urls-for-bundle-components), which is
```

File: explainers/navigation-to-unsigned-bundles.md [hunk 5]
```diff
@@ -237,6 +230,115 @@ in the bundle, there are several options:
...
+#### 14. How does this specification work in the context of a user agent’s Private Browsing or "incognito" mode?
+
+No difference while the user is browsing sites in Private mode, except that any
+session state will be persisted (in the same way as regular navigation in
+Private mode).
+
+#### 15. Does this specification have a "Security Considerations" and "Privacy Considerations" section?
+
+See the security considerations in the format spec, security and privacy
+considerations in the explainer, and open design questions in the explainer.
```
Comment (horo-t) [inline explainers/navigation-to-unsigned-bundles.md]:
```suggestion
See the [security considerations](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#security)
in the format spec, [security and privacy considerations](#security-and-privacy-considerations)
in the explainer, and [open design questions](#open-design-questions) in the
explainer.
```

**Comment by kinu** [comment]
Still lgtm.

**Pull request merged**